spec/policies/organisation_policy_spec.rb

```diff
@@ -16,13 +16,21 @@
   end
 
   context "as a delivery partner" do
-    let(:user) { build_stubbed(:user) }
+    let(:user) { build_stubbed(:delivery_partner) }
 
     it { is_expected.to permit_action(:index) }
-    it { is_expected.to permit_action(:show) }
+    it { is_expected.to forbid_action(:show) }
 
     it { is_expected.to forbid_new_and_create_actions }
     it { is_expected.to forbid_edit_and_update_actions }
     it { is_expected.to forbid_action(:destroy) }
+
+    context "when the delivery_partner belongs to that organisation" do
+      let(:user) { build_stubbed(:delivery_partner, organisations: [organisation]) }
+      let(:organisation) { create(:organisation) }
+
+      it { is_expected.to permit_action(:index) }
+      it { is_expected.to permit_action(:show) }
+    end
```

Comment:
This file is missing scope tests since `policy_scope` is called in the controller.